internal/cli/main.go

```diff
@@ -227,6 +227,16 @@ Database drivers: `+strings.Join(database.List(), ", ")+"\n")
 		}
 
 	case "drop":
+		log.Println("Are you sure you want to drop the entire database? [y/N]")
+		var response string
+		fmt.Scanln(&response)
+		response = strings.ToLower(strings.TrimSpace(response))
+
+		if response != "y" {
```

Comment:
Use the pattern in `down`. e.g. provide feedback that the request action is being performed

Reply:
All done - I also slightly changed the message to say "drop the database schema" instead of "drop the database" because they are different things.